### PR TITLE
Adds pricing page and status links to footer

### PR DESCRIPTION
**Pull request by daniellemaxwell**
Scope of Changes:

Adds links to pricing page and status in the footer.

Fixes SC-22996

Acceptance Criteria:
https://www.awesomescreenshot.com/image/44885421?key=22e342d54d2669b817215f46057e5b05

### CODE REVIEW

File: data/en/footer.yml [hunk 1]
```diff
@@ -8,10 +8,15 @@ footer:
...
     - name: Ensign
       link: https://rotational.app
-    - name: Documentation
+    - name: Pricing
+      link: /ensign-pricing
+    - name: Docs
       link: https://ensign.rotational.dev/getting-started/
     - name: SDKs
       link: https://ensign.rotational.dev/sdk/
+    - name: Status
```
Comment (daniellemaxwell) [inline data/en/footer.yml:17]: `Status` was added to maintain consistency with the rotational.app footer.

File: data/en/footer.yml [hunk 2]
```diff
@@ -8,10 +8,15 @@ footer:
     product_items:
     - name: Ensign
       link: https://rotational.app
-    - name: Documentation
+    - name: Pricing
+      link: /ensign-pricing
+    - name: Docs
```
Comment (daniellemaxwell) [inline data/en/footer.yml:13]: `Documentation` looked really long after adding `Pricing`. Shortening the word to `Docs` makes it more in line with the others and it's common to see `Docs` used on other sites.

Comment (pdeziel) [reply data/en/footer.yml:13]: Makes sense to me!

**Review by pdeziel — APPROVED** [review]
Looks good! One thing I noticed was that on rotational.app the footer links are center-aligned, while the links on rotational.io are left-aligned. I'm not sure if that was intentional and I don't have a strong opinion on which way is better, I just wanted to point that out in case we want to make those more consistent in the future.

**Pull request merged**